domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -1124,9 +1150,11 @@ class ExplorationProgressController @Inject constructor(
             NEXT_AVAILABLE_HINT_INDEX -> // No solution, so revealing the hint ends available help.
               stateAnalyticsLogger?.logViewHint(helpIndex.nextAvailableHintIndex)
             // Nothing to do in these cases.
-            LATEST_REVEALED_HINT_INDEX, EVERYTHING_REVEALED, INDEXTYPE_NOT_SET, null -> {}
+            LATEST_REVEALED_HINT_INDEX, EVERYTHING_REVEALED, INDEXTYPE_NOT_SET, null -> {
+            }
           }
-          INDEXTYPE_NOT_SET, null -> {} // Nothing to do here.
+          INDEXTYPE_NOT_SET, null -> {
+          } // Nothing to do here.
```

Comment:
Here & elsewhere in this file: please revert the format-only changes.

Reply:
Still relevant--please resolve this. There are multiple cases in this file, so it's probably worth checking the whole PR just to make sure no style-only changes are added.

Reply:
done